src/v2/ion/uiSchemaTransformer.js

```diff
@@ -173,45 +173,79 @@ const createUISchema = (transformedResp, remediationForm) => {
     })
     .flatten()
     .value();
+
+  const getCheckboxUiSchema = (ionFormField, uiSchema) => {
```

Comment:
@haishengwu-okta , looks like there are new lint rules (only 15 statements allowed in a function or an arrow function), hence extracted them to their own granular functions.

Reply:
that's because `  /* eslint complexity: [2, 14] */` at line 157.
my intention was let it grow and find chance to refactor later.
I'm fine with 
- either your changes but your `get***` function shall return pure object instead change `options` in place. otherwise it's unconventional.
- or update the eslint override

Reply:
Updated with some ES6 sugar. Nevertheless, needed to update the complexity as well.